ci-operator/config/openshift/lws-operator/openshift-lws-operator-main.yaml

```diff
@@ -1,5 +1,5 @@
 base_images:
-  ocp_builder_base-rhel9:
+  ocp_4.19_base-rhel9:
     name: builder
```

Comment:
I think these have to be changed to `4.19` Please see https://github.com/openshift/release/pull/61575 and e.g. https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_jobset-operator/1/pull-ci-openshift-jobset-operator-main-fips-image-scan-jobset-operator/1889688802629259264

Reply:
hm, this should have merged already: https://github.com/openshift/release/pull/61493/files

Reply:
Yeah, hold couldn't catch the merge pool, I'm opening a new PR now. Thanks for heads up.

Reply:
Opened https://github.com/openshift/release/pull/61576

Reply:
This was caused by the bot: https://github.com/openshift/release/pull/61526. I wonder why.

Reply:
hmm, because master https://github.com/openshift/lws-operator/blob/c1c6c4bf5ec6eca421b2a57ae2cd937734d9d773/Dockerfile.ci#L6 hasn't been updated yet. We need to merge operator PRs. Otherwise, this keep going.

Reply:
I see, bots are faster than us :D